src/theme/book.js

```diff
@@ -262,8 +255,79 @@ $( document ).ready(function() {
     clipboardSnippets.on('error', function(e) {
             showTooltip(e.trigger, "Clipboard error!");
     });
+
+    $.ajax({
+        url: "https://play.rust-lang.org/meta/crates",
+        method: "POST",
+        crossDomain: true,
+        dataType: "json",
+        contentType: "application/json",
+        success: function(response){
```

Comment:
What do we do on error?

Reply:
@azerupi  Currently nothing. The play buttons will not appear which imho is a safer default than showing play buttons on content that might be unsupported by play.rlo. 
I did not have a reasonable fallback scenario, do you have anything in mind?

Reply:
No, that is perfect :+1: 
Not showing the play button when the playground can not be reached makes a lot of sense :)